v2.0/paths/task_assignments__get_tasks_id_assignments.yml

```diff
@@ -23,3 +23,10 @@ responses:
       application/json:
         schema:
           $ref: '#/components/schemas/TaskAssignments'
+  500:
```

Comment:
Does this really return a TaskAssignments object? And should we really document 500s?

Reply:
I think we should document this 500, but I don't generally think we should document 500s. And no, it doesn't I revised it to remove that.

Reply:
This **should** be a 404 or some other error than a 500, but until that is fixed in the API, some users may encounter it in the current state.

Reply:
Can you highlight to me when this 500 happens?

Reply:
![image](https://user-images.githubusercontent.com/9373485/60297580-21e7f980-98ee-11e9-9b54-13f36fe49966.png)
I believe I got a different message in a past request, but I didn't screenshot it 😞

Reply:
So, technically this isn’t calling that path. `task_assignment_id` is not nullable. I understand that it might be an error, but it’s technically documenting a different endpoint (that probably actually exists, just not as an official endpoint). 

Regardless, we should either document this with a clearer message of when this happens (“Returns an error if the `task_assignment` ID is omitted.) or not document it at all.

Reply:
We still need to add the response body here. Same ClientError format I think